src/avram/database.cr

```diff
@@ -158,6 +158,12 @@ abstract class Avram::Database
     end
   end
 
+  # Close all available connections as well as the DB
+  def self.close_connections
+    connections.values.map(&.close)
+    @@db.try(&.close)
+  end
```

Comment:
Not really a fan of this... I'm basically just saying "grab everything and close it all". I'd assume if you ran this in your app, everything would go really bad :grimacing: